src/redis-cli.c

```diff
@@ -7650,7 +7651,7 @@ static void findBigKeys(int memkeys, unsigned memkeys_samples) {
         }
 
         /* Sleep if we've been directed to do so */
-        if(sampled && (sampled %100) == 0 && config.interval) {
+        if (scan_loops && (scan_loops % 100) == 0 && config.interval) {
```

Comment:
Checking `config.interval` first is a slight optimization to avoid the theoretically costly modulus. Also I think because of the `scan_loop++` at the top there's no need to verify it isn't 0 on the first iteration.
```suggestion
        if (config.interval && scan_loops % 100 == 0) {
```

Reply:
@yoav-steinberg Comments addressed. PTAL.